lib/aws/active_job/sqs/configuration.rb

```diff
@@ -86,13 +114,22 @@ class Configuration
         #   Using this option, job_id is implicitly added to the keys.
 
         def initialize(options = {})
-          options[:config_file] ||= config_file if File.exist?(config_file)
-          options = DEFAULTS
-                    .merge(file_options(options))
-                    .merge(options)
-          set_attributes(options)
+          opts = env_options.deep_merge(options)
```

Comment:
Check where deep merge is coming from, but if you pull rails out, you may not have this anymore

Reply:
its active support, which is reasonable to keep as a dependency I think.

Otherwise we'll need to define our own. Thoughts?

Reply:
If active job by itself depends on it then that's fine. If it doesn't we should write our own